src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpConnection.cs

```diff
@@ -102,7 +102,9 @@ public HttpConnection(Socket sock, HttpEndPointListener epl, bool secure, X509Ce
 
             _timer = new Timer(OnTimeout, null, Timeout.Infinite, Timeout.Infinite);
             if (_sslStream != null) {
+#pragma warning disable SYSLIB0014 // ServicePointManager is obsolete
                 _sslStream.AuthenticateAsServer (_cert, true, (SslProtocols)ServicePointManager.SecurityProtocol, false);
```

Comment:
This is in the Unix implementation.  What does the Windows implementation use?  If it's not referencing the same thing, is that a bug we should fix?

Reply:
I don't know much about HttpListener, but I would assume on Windows we delegate functionality to `httpapi` at a higher level. /cc @wfurt